### ISSUE TITLE
Make clearer that "preview..." is on the path to "send and publish"

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#6242]
Occasionally users get stuck when faced with only the option to 

>"Preview your public request"

They don't realise that button will take them down the path towards sending and publishing their request. 

A user who reached this point wrote to WhatDoTheyKnow to say

>I have filled in my reply in the box provided but I cannot find how to actually send it.  Is there a box I should click or button I should press?

Perhaps we need to say something like "Next step: Preview your public request" ... or "Step 2/3: Preview your public request"

Perhaps the steps to be followed should be all shown then ticked off as progressed?

This issue comes up on occasion and was mentioned at:

https://github.com/mysociety/alaveteli/issues/3585#issuecomment-257057207

**Comment by garethrees** [comment]
Quickest win here is just changing the button text to "Next step: Preview your public request".

![Screenshot 2021-05-12 at 16 27 29](https://user-images.githubusercontent.com/282788/118001850-039f1180-b33f-11eb-9211-e4725726fbbe.png)

* "Step 2/3: Preview your public request" might not easily work across all the private/batch/public interfaces.
* Checklist of steps would be nice but harder to maintain and match up where someone ise.
